code/datums/mutations/actions.dm

```diff
@@ -167,10 +167,10 @@
 	energy_coeff = 1
 	synchronizer_coeff = 1
 
-/datum/mutation/human/void/on_life()
+/datum/mutation/human/void/on_life(delta_time, times_fired)
 	if(!isturf(owner.loc))
 		return
-	if(prob((0.5+((100-dna.stability)/20))) * GET_MUTATION_SYNCHRONIZER(src)) //very rare, but enough to annoy you hopefully. +0.5 probability for every 10 points lost in stability
+	if(DT_PROB((0.25+((100-dna.stability)/40)) * GET_MUTATION_SYNCHRONIZER(src), delta_time)) //very rare, but enough to annoy you hopefully. +0.5 probability for every 10 points lost in stability
```

Comment:
Fix probability, might be abit more difficult here but perhaps tune the 0.25 and /40 to better adapt to common dna.stability values

Reply:
The issue here is that common dna stability values are 0-100

Reply:
I did the math and the maximum difference between the two is about 0.03%
I'm not going to bother with a more complicated calculation.